occurrences/tests/snapshots/snap_test_api.py

```diff
@@ -295,7 +295,7 @@
                         {
                             "node": {
                                 "id": "grade_1",
-                                "label": None,
+                                "label": "first grade",
```

Comment:
I don't know why these are in this PR :D Should they be or should they not?

Reply:
They should belong to the update translations of study levels PR I guess. But let's just keep it here, otherwise the test will fail